Software-Engineering.md

```diff
@@ -133,6 +133,24 @@ _We avoid distinguishing too much between the Junior Software Engineer and Softw
 - I can confidently and charismatically pitch ideas, positively influencing my team to take decisive action based on reaching consensus.
 - I am an adept communicator, and can effectively steer technical and non-technical conversations to positive outcomes over any medium.
 
+## Principal Software Engineer
+
+> I am an influential Engineering [leader](Leadership.md) who identifies problems and opportunities relevant to Octopus' strategy, builds consensus to invest in solutions, and guides Engineering to deliver significant lasting change across Octopus.
+
+### I have a knack for uncovering important problems and opportunities that are critical to the long-term success of Octopus
+
+- I have a deep understanding of Octopus products and our strategy
+- I identify significant problems, inhibitors and opportunities, and advocate for solutions that align our strategy and Engineering efforts
+- I collaborate with others to articulate solutions, positively influence and convince that we should fund their implementation
+- My contributions go beyond patching or extending existing solutions, which may not be fit for purpose anymore. They open up whole new areas of strategic possibility, to take our Engineering team, and Octopus as a whole, to the next level.
```

Comment:
Is there a better way to say `to the next level`?

Reply:
@andrewabest will wordsmith. :)

Paul loves "world-class".

Reply:
```suggestion
- My contributions go beyond patching or extending existing solutions, which may not be fit for purpose anymore. They open up whole new areas of strategic possibility that progress our journey of making Octopus a world-class Engineering organization.
```